spec/outputs/dynatrace_spec.rb

```diff
@@ -232,18 +232,39 @@
     end
   end
 
-  context 'max_payload_size 2MB' do
-    let(:config) { { 'ingest_endpoint_url' => ingest_endpoint_url, 'api_key' => api_key, 'max_payload_size' => 2_000_000 } }
+  context 'max_payload_size 500' do
```

Comment:
Can we add a test case that does not only look at the number of exported requests but also ensures the byte size is at most max_payload_size even when getting close to the max?

Reply:
ecebba7de1a02d2164f3c6d261b9c68170780dd7